tools/shared/src/main/scala/org/scalajs/core/tools/linker/frontend/optimizer/OptimizerCore.scala

```diff
@@ -3451,49 +3451,16 @@ private[optimizer] abstract class OptimizerCore(
           case _ => default
         }
 
-      case Float_+ =>
-        (lhs, rhs) match {
-          case (PreTransLit(FloatLiteral(0)), _) =>
-            rhs
-          case (_, PreTransLit(FloatLiteral(_))) =>
-            foldBinaryOp(Float_+, rhs, lhs)
```

Comment:
These two are also incorrect? Or just not worth it anymore? (similar comments below).

Reply:
The first one is incorrect : `+0.0f + -0.0f` gives `+0.0f`, not `rhs` which is `-0.0f`.

The second one is not worth it anymore.